src/main/java/uk/gov/justice/digital/hmpps/whereabouts/controllers/AppointmentController.kt

```diff
@@ -30,15 +36,37 @@ class AppointmentController(private val appointmentService: AppointmentService)
         code = 404,
         message = "Appointment not found.",
         response = ErrorResponse::class,
-        responseContainer = "List"
       ),
       ApiResponse(
         code = 500,
         message = "Unrecoverable error occurred whilst processing request.",
         response = ErrorResponse::class,
-        responseContainer = "List"
       )
     ]
   )
   fun getAppointment(@PathVariable("id") id: Long): AppointmentDetailsDto = appointmentService.getAppointment(id)
+
+  @PostMapping
+  @ResponseStatus(HttpStatus.CREATED)
+  @ApiOperation(
+    value = "Create an appointment",
+    nickname = "createAppointment"
+  )
+  @ApiResponses(
+    value = [
+      ApiResponse(code = 201, message = "CREATED", response = AppointmentCreatedDto::class),
```

Comment:
not sure this line is needed, swagger should really be able to work it out (well it can for 200 responses anyway)